documentation/en/cli-lotus-miner.md

```diff
@@ -1884,44 +1883,8 @@ OPTIONS:
    
 ```
 
-### lotus-miner sectors renew
+#### lotus-miner sectors renew, extend
```

Comment:
Looks like the cli docsgen script is not happy with aliases..  oh well

Reply:
Okay, alias has been removed.